docs/dummies.md

```diff
@@ -63,8 +63,8 @@ argument values. It uses Dummies.
 When FakeItEasy needs to access a Dummy of type `T`, it tries a number
 of approaches in turn, until one succeeds:
 
-1. If there's a user-supplied
-  [custom Dummy creation](custom-dummy-creation.md) mechanism for `T`,
+1. If there's a user-supplied or built-in
+  [dummy factory](custom-dummy-creation.md) for `T`,

```

Comment:
I don't know if we should mention that we have built-in dummy factories. It's more accurate like this, but maybe the user doesn't need to know. It's a separate commit, so I can easily drop it if you prefer.

Reply:
Or maybe we should just add an item in the list for `String`, which is effectively a special case.

Reply:
I wouldn't mention it. There's no user-visible consequence.

Reply:
Oops. I commented on another page where I didn't see the `String` comment. I would support mentioning `String` (but don't feel it's _necessary_), but don't think there's any point in mentioning build-in Dummy factories.

Reply:
OK, I'll just add a few words about `String`